tools/docker/Dockerfile.ubuntu-prod

```diff
@@ -21,7 +21,7 @@ FROM ubuntu:20.04
 ARG QEMU_CPU
 ARG DEBIAN_FRONTEND=noninteractive
 
-RUN apt clean && apt update && apt -y install netcat-openbsd ca-certificates redis-tools libxml2
+RUN apt clean && apt update && apt -y install netcat-openbsd ca-certificates redis-tools libxml2 bison
```

Comment:
Is this not needed? 🤔

Reply:
Nope, bison is only needed at build time, it doesn't have any runtime libraries (at least not that we use)